api/model.go

```diff
@@ -41,6 +41,7 @@ type Volume struct {
 	State                            longhorn.VolumeState                   `json:"state"`
 	Robustness                       longhorn.VolumeRobustness              `json:"robustness"`
 	EngineImage                      string                                 `json:"engineImage"`
+	Image                            string                                 `json:"image"`
```

Comment:
Can we just remove `EngineImage` file here? It seems it will be invalid after upgrade, right?

Do we need UI part to do the corresponding change?

Reply:
Removed the `EngineImage`, thank you.

> Remove deprecated fields from CRDs in v1.7.0

Yes, tracked in https://github.com/longhorn/longhorn/issues/6685